Improve the database audit log feature (#11264)
pingcap/docs: tidb-cloud/tidb-cloud-auditing.md

> 

Reviewer: 这次 GA 的只有 database audit logging，不包括 console audit logging 是吗？

Author: 是的，这是两个功能点，独立的

TiDB Cloud provides you with a database audit logging feature to record a history of user access details (such as any SQL statements executed) in logs.

> 

Reviewer 2: @ljun0712  Since this feature is GA now, please also add the link of this doc to the `TOC-tidb-cloud.md` file so users can find this doc in the document left navigation pane. Thanks.

If you no longer want to audit a cluster, go to the page of the cluster, click **Settings** > **Audit Settings**, and then toggle the audit setting in the upper-right corner to **Off**.

> **Note:**
>

> > Each time the log file is filled with 10 MiB, it will be pushed to the cloud storage bucket. Therefore, after the audit log is disabled, the log file smaller than 10 MiB will not be automatically pushed to the cloud storage bucket. If you need to get this log file, contact [PingCAP support](/tidb-cloud/tidb-cloud-support.md).

Reviewer 3: ```suggestion
> Each time the size of the log file reaches 10 MiB, the log file will be pushed to the cloud storage bucket. Therefore, after the audit log is disabled, the log file size smaller than 10 MiB will not be automatically pushed to the cloud storage bucket. To get the log file in this situation, contact [PingCAP support](/tidb-cloud/tidb-cloud-support.md).
```

> > Because the log file will be pushed to the cloud storage bucket when it is filled with 10M each time. Therefore, when the audit log is disabled, the log file less than 10M will not be automatically pushed to the cloud storage bucket. If you need to get this log file, you can contact [PingCAP support team](/tidb-cloud/tidb-cloud-support.md).

Reviewer: ```suggestion
> Each time the log file is filled with 10 MiB, it will be pushed to the cloud storage bucket. Therefore, after the audit log is disabled, the log file smaller than 10 MiB will not be automatically pushed to the cloud storage bucket. If you need to get this log file, contact [PingCAP support](/tidb-cloud/tidb-cloud-support.md).
```

Reviewer: @TomShawn PTAL

If you no longer want to audit a cluster, go to the page of the cluster, click **Settings** > **Audit Settings**, and then toggle the audit setting in the upper-right corner to **Off**.

> **Note:**
>

> > After the audit log is disabled, the last log file will not be pushed to the bucket on the cloud. You can push this file in two ways:

Reviewer 3: ```suggestion
> After the audit log is disabled, the last log file will not be pushed to the bucket on the cloud. You can push this file in either of the following ways:
```

> > After the audit log is disabled, the last log file will not be pushed to the bucket on the cloud. You can push this file in two ways:

Reviewer 4: just clarify that the log will and will only be pushed once it reaches size 10M.
how can the user restart a cluster?, scale-out not work

If you no longer want to audit a cluster, go to the page of the cluster, click **Settings** > **Audit Settings**, and then toggle the audit setting in the upper-right corner to **Off**.

> **Note:**
>

> > After the audit log is disabled, the last log file will not be pushed to the bucket on the cloud. Users can push this file to the bucket on the cloud in two ways:
> > - You can restart the cluster, or expand the cluster.
> > - You can contact the PingCAP support team.

Reviewer: ```suggestion
> After the audit log is disabled, the last log file will not be pushed to the bucket on the cloud. You can push this file in two ways:
>
> - Restart the cluster, or scale out the cluster.
> - Contact the PingCAP support team.
```
